diff --git a/src/generated/Modulos/Controle-NF-Entrada/Controle NF Entrada.ps1 b/src/generated/Modulos/Controle-NF-Entrada/Controle NF Entrada.ps1
--- a/src/generated/Modulos/Controle-NF-Entrada/Controle NF Entrada.ps1
+++ b/src/generated/Modulos/Controle-NF-Entrada/Controle NF Entrada.ps1
@@ -8,2 +8,3 @@
 Add-Type -AssemblyName System.Windows.Forms
 Add-Type -AssemblyName System.Drawing
+Add-Type -AssemblyName Microsoft.VisualBasic
