diff --git a/src/generated/Modulos/Controle-NF-Entrada/Controle NF Entrada.ps1 b/src/generated/Modulos/Controle-NF-Entrada/Controle NF Entrada.ps1
--- a/src/generated/Modulos/Controle-NF-Entrada/Controle NF Entrada.ps1
+++ b/src/generated/Modulos/Controle-NF-Entrada/Controle NF Entrada.ps1
@@ -1577,1 +1577,1 @@
-$movementFooter.Dock=[Windows.Forms.DockStyle]::Fill; $movementFooter.ColumnCount=2
+$movementFooter.Dock=[Windows.Forms.DockStyle]::Fill; $movementFooter.ColumnCount=3
